### PR TITLE
Updated Courtlistener header image, OG images and CSS colors.

### PR DESCRIPTION
**Pull request by albertisfu**
Here are some screenshots showing how the new logo appears on different screen sizes:

Screenshot 2024-07-05 at 3 45 16 p m
Screenshot 2024-07-05 at 3 45 03 p m
Screenshot 2024-07-05 at 3 44 48 p m

The full logo version has the link to `free.law` embedded into the SVG.
The height of the logo on desktop is 65px. Making it smaller makes the "By FREE.LAW" part hard to read.

- Also the old red color in the CSS was replaced by the new `#B53C2C`

- Replaced the main OG image: [og-image-1280x640.png](https://github.com/freelawproject/courtlistener/compare/new-logo-courtlistener?expand=1#diff-93c962946f65fa106d8242e2b09aaf178c98e5001adbb7ee2f1e341cfe827b11)

- And the one for Tags: [og-image-tag-300x300.png](https://github.com/freelawproject/courtlistener/compare/new-logo-courtlistener?expand=1#diff-65a00930e0b50446aff7c92ad912cff7abecced7243d3ce3af67b7bab95bafb9)

 I'll open a different PR to update the favicon once we have the final files.

### CODE REVIEW

File: cl/assets/templates/base.html [hunk 1]
```diff
@@ -165,13 +165,10 @@ <h1>You did not supply the "private" variable to your template.
             <span class="icon-bar"></span>
           </button>
           <a class="navbar-brand hidden-xs" href="/">
-            <img src="{% static "svg/logo-full.svg" %}"
-                 alt="CourtListener"
-                 height="40"
-                 tabindex="1"/>
+            {% include "includes/cl_logo_svg.html" %}
```
Comment (mlissner) [inline cl/assets/templates/base.html:168]: I think you can name the file whatever you want and do:

```suggestion
            {% include "includes/cl_logo.svg" %}
```

Pretty sure there's no requirement to include only `html` files.

File: cl/assets/templates/base.html [hunk 2]
```diff
@@ -165,13 +165,10 @@ <h1>You did not supply the "private" variable to your template.
             <span class="icon-bar"></span>
           </button>
           <a class="navbar-brand hidden-xs" href="/">
```
Comment (mlissner) [inline cl/assets/templates/base.html:167]: Can we move the tabindex here?

File: cl/assets/static-global/png/og-image-1280x640.png
Comment (mlissner) [inline cl/assets/static-global/png/og-image-1280x640.png:1]: We should either make this the dimensions of the old file or we should update the name of the file. I don't think the dimensions matter much between the two, so whatever you like is fine.

**Review by mlissner — CHANGES_REQUESTED** [review]
A couple little things, thank you!

File: cl/assets/templates/base.html (hunk 1 above)
Comment (albertisfu) [reply cl/assets/templates/base.html]: Yeah it works well using the SVG extension, changed it.

File: cl/assets/templates/base.html (hunk 2 above)
Comment (albertisfu) [reply cl/assets/templates/base.html]: Sure, fixed.

File: cl/assets/static-global/png/og-image-1280x640.png
Comment (albertisfu) [reply cl/assets/static-global/png/og-image-1280x640.png:1]: Yeah, I updated the file name to `og-image-1200x630.png`

File: cl/assets/templates/base.html (hunk 1 above)
Comment (quevon24) [reply cl/assets/templates/base.html]: i was poking around and found this very useful 🕵️

Comment (mlissner) [reply cl/assets/templates/base.html]: We use this approach a lot on bots.law.

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
Here we go!!

**Pull request merged**